aptos-move/framework/aptos-framework/sources/AptosLib.move

```diff
@@ -0,0 +1,29 @@
+module AptosFramework::AptosLib {
+    struct TypeInfo has copy, drop {
+        account_address: address,
+        module_name: vector<u8>,
+        struct_name: vector<u8>,
+    }
+
+    public fun type_info_address(type_info: &TypeInfo): address {
```

Comment:
why not give `TypeInfo` its own module? `TypeInfo::address()` is so much better.

Reply:
approve the PR